### PR TITLE
Feature merchant/price routes

### PR DESCRIPTION
**Pull request by Kiwi-x-Kiwi**

### CODE REVIEW

File: server/routes/merchants.js [hunk 1]
```diff
@@ -0,0 +1,120 @@
...
+ * Delete a specific merchant
+ * @example 
+ * DELETE /api/merchants/:id
+ */
+router.delete('/:id', isLoggedIn, async (req, res) => {
+  try {
+    const merchant = await Merchant.findById(req.params.id)
+    if (!merchant) throw new Error()
+    if (req.user.role !== "admin") res.status(403).send('You do not have permission to delete this resource.')
+    Price.deleteMany({ merchant: merchant._id })
```
Comment (AndresMWeber) [inline server/routes/merchants.js:81]: There's a better way to do this:
In the userSchema you can add a static method that runs on "delete" and can delete any related prices that way.

**Review by AndresMWeber — APPROVED** [review]

File: server/routes/merchants.js [hunk 2]
```diff
@@ -0,0 +1,120 @@
+const express = require('express')
+const { isLoggedIn } = require('../middlewares')
+const Price = require('../models/Price')
+const Merchant = require("../models/Merchant")
```
Comment (bobbypwang) [inline server/routes/merchants.js:4]: Lets keep things consistent. Try to use single quotes.

**Review by bobbypwang — APPROVED** [review]

File: server/routes/merchants.js (hunk 1 above)
Comment (Kiwi-x-Kiwi) [reply server/routes/merchants.js:81]: 👍 Looks a lot cleaner that way.  Updated.

File: server/routes/merchants.js (hunk 2 above)
Comment (Kiwi-x-Kiwi) [reply server/routes/merchants.js:4]: Got it, will try to stick to single quotes from here on out.  Thanks for the input!

**Pull request merged**